src/components/molecules/Settings/Workspace/Asset/AssetDeleteModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import Text from "@reearth/components/atoms/Text";
+import Modal from "@reearth/components/atoms/Modal";
+import Divider from "@reearth/components/atoms/Divider";
+import Button from "@reearth/components/atoms/Button";
+
+import { styled } from "@reearth/theme";
+import { metricsSizes } from "@reearth/theme/metrics";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface Props {
+  isVisible: boolean;
+  onClose?: () => void;
+  handleRemove?: () => void;
+}
+
+const AssetDeleteModal: React.FC<Props> = ({ isVisible, onClose, handleRemove }) => {
+  const intl = useIntl();
+  return (
+    <Modal
+      title="Delete assets"
+      isVisible={isVisible}
+      size="sm"
+      onClose={onClose}
+      button1={
+        <Button
+          text={intl.formatMessage({ defaultMessage: "Delete" })}
+          buttonType="danger"
+          onClick={handleRemove}
+        />
+      }
+      button2={
+        <Button
+          text={intl.formatMessage({ defaultMessage: "Cancel" })}
+          buttonType="secondary"
+          onClick={onClose}
+        />
+      }>
+      <Divider margin="0" />
+      <Message size="m">
+        {intl.formatMessage({
+          defaultMessage:
+            "You are about to delete assets from the workspace. Any place which use this asset will cause a error. And this is an action that cannot be undone.",
+        })}
+      </Message>
+      <Message size="m">
+        {intl.formatMessage({
+          defaultMessage: "Are you sure you would like to delete these assets?",
```

Comment:
Maybe the above message is enough? If you think this should be here, maybe "Are you sure you would like to delete the selected asset(s)?"

Reply:
Yes, I hope we can keep this I will fix